Unknown Exception occurs when invalid AR~ token is given

## Environment
- Core Version: v1.7.20

## Reproduction procedure
Send  a Http request with invalid AR~ token.

```console
$ curl -X GET -k -i -H "Authorization: Bearer AR~A"  https://app-timefiller.appdev.personium.io/__/launch.json
HTTP/1.1 200 Connection established

HTTP/1.1 500 
Date: Thu, 21 Nov 2019 07:20:29 GMT
Content-Length: 87
Connection: keep-alive
X-Personium-Version: 1.7.20
Server: Personium

{"code":"PR500-SV-9999","message":{"lang":"en_US","value":"Unknown Exception [35234]"}}
```

## personium-core.log

```log
2019-11-21 16:31:29.707 [catalina-exec-20] [INFO ] PersoniumCoreContainerFilter [1.7.20] Started. GET http://app-timefiller.appdev.personium.io/personium-core/__/launch.json *.*.*.*
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter == Reqeust Headers
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter x-forwarded-for:*.*.*.*
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter x-forwarded-proto:https
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter x-forwarded-path:/__/launch.json
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter host:app-timefiller.appdev.personium.io
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter connection:close
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter user-agent:curl/7.51.0
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter accept:*/*
2019-11-21 16:31:29.708 [catalina-exec-20] [DEBUG] PersoniumCoreContainerFilter authorization:Bearer AR~A
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource Call API 'https://app-timefiller.appdev.personium.io/__/launch.json'.
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource     p_cookie: null
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource     p_cookie_peer: null
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource     Authorization: Bearer AR~A
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource     X-Personium-Unit-User: null
2019-11-21 16:31:29.709 [catalina-exec-20] [DEBUG] FacadeResource     X-Personium-RequestKey: null
2019-11-21 16:31:29.715 [catalina-exec-20] [DEBUG] FacadeResource     X-Personium-EventId: null
2019-11-21 16:31:29.715 [catalina-exec-20] [DEBUG] FacadeResource     X-Personium-RuleChain: null
2019-11-21 16:31:29.715 [catalina-exec-20] [DEBUG] FacadeResource     X-Personium-Via: null
2019-11-21 16:31:29.715 [catalina-exec-20] [INFO ] PersoniumCoreLog [PL-SV-0022] - RequestKey generated: [PCS-03ca2fdf-93e7-4614-84fe-a203f4ebfe58] - [io.per
sonium.core.rs.FacadeResource#facade:101]
2019-11-21 16:31:29.739 [catalina-exec-20] [ERROR] PersoniumCoreExceptionMapper [PR500-SV-9999] - Unknown Exception [62073] 6 @io.personium.common.auth.token
.AbstractLocalToken#doParse: 173
java.lang.ArrayIndexOutOfBoundsException: 6
        at io.personium.common.auth.token.AbstractLocalToken.doParse(AbstractLocalToken.java:173)
        at io.personium.common.auth.token.AbstractLocalToken.populate(AbstractLocalToken.java:181)
        at io.personium.common.auth.token.ResidentLocalAccessToken.parse(ResidentLocalAccessToken.java:98)
        at io.personium.common.auth.token.AbstractOAuth2Token.parse(AbstractOAuth2Token.java:307)
        at io.personium.core.auth.AccessContext.createBearerAuthz(AccessContext.java:752)
        at io.personium.core.auth.AccessContext.create(AccessContext.java:222)
        at io.personium.core.rs.FacadeResource.facade(FacadeResource.java:123)
        at sun.reflect.GeneratedMethodAccessor60.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.glassfish.jersey.server.internal.routing.SubResourceLocatorRouter.lambda$getResource$0(SubResourceLocatorRouter.java:156)
        at org.glassfish.jersey.server.internal.routing.SubResourceLocatorRouter.getResource(SubResourceLocatorRouter.java:174)
        at org.glassfish.jersey.server.internal.routing.SubResourceLocatorRouter.apply(SubResourceLocatorRouter.java:110)
        at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:110)
        at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:113)
        at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:113)
        at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:113)
        at org.glassfish.jersey.server.internal.routing.RoutingStage._apply(RoutingStage.java:113)
        at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:93)
        at org.glassfish.jersey.server.internal.routing.RoutingStage.apply(RoutingStage.java:62)
        at org.glassfish.jersey.process.internal.Stages.process(Stages.java:197)
        at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:269)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:272)
        at org.glassfish.jersey.internal.Errors$1.call(Errors.java:268)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:316)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:298)
        at org.glassfish.jersey.internal.Errors.process(Errors.java:268)
        at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:289)
        at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:256)
        at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:703)
        at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:416)
        at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:370)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:389)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:342)
        at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:229)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:491)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:764)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1388)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:748)
2019-11-21 16:31:29.745 [catalina-exec-20] [INFO ] PersoniumCoreContainerFilter [1.7.20] Completed. 500 37ms
```

## Case that is not Unknown Exception

```
$ curl -X GET -i -H "Authorization: Bearer AR~AB"  https://app-timefiller.appdev.personium.io/__/
launch.json
HTTP/1.1 200 Connection established

HTTP/1.1 200 
Date: Thu, 21 Nov 2019 07:23:58 GMT
Content-Type: application/json
Content-Length: 366
Connection: keep-alive
ETag: "15-1571379204844"
Accept-Ranges: bytes
X-Personium-Version: 1.7.20
Server: Personium

{
  "personal": {
     "web": "https://app-timefiller.appdev.personium.io/__/html/index.html",
     "android": "***:",
     "ios": "***:",
     "sendRefreshToken": true
  },
  "organization": {
         "web": "https://app-timefiller.appdev.personium.io/__/html/index_org.html",
     "android": "***:",
     "ios": "***:",
     "sendRefreshToken": true
  }
}
```

--- a/pom.xml
+++ b/pom.xml
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>io.personium</groupId>
             <artifactId>personium-lib-common</artifactId>
-            <version>1.5.2-SNAPSHOT</version>
+            <version>1.5.3-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>io.personium</groupId>

